src/devices/esp8266.c

```diff
@@ -329,6 +330,9 @@ bool esp8266_init(struct Serial *s, const size_t max_cmd_len,
         if (!_setup(s, max_cmd_len))
                 return false;
 
+        /* Hard reset our ESP8266*/
```

Comment:
gotcha. want to leave it here until #583 is implemented? That way we at least have proper reset control with RCT.